frontend/src/components/Hero.tsx

```diff
@@ -8,14 +11,26 @@ const Hero = () => {
 
   return (
     <div data-testid="hero" className="usa-dark-background bg-primary">
-      <GridContainer className="padding-y-1 tablet:padding-y-3 tablet-lg:padding-y-10 desktop-lg:padding-y-15">
+      <GridContainer className="padding-y-1 tablet:padding-y-3 tablet-lg:padding-y-10 desktop-lg:padding-y-15 position-relative">
         <h1 className="tablet:font-sans-2xl desktop-lg:font-sans-3xl text-ls-neg-2">
           <span>{t("title")}</span>
         </h1>
         <p className="usa-intro font-sans-md tablet:font-sans-lg desktop-lg:font-sans-xl">
           <span className="text-yellow text-bold">{t("beta")}</span>&nbsp;
           {t("content")}
         </p>
+        <Link
```

Comment:
Not really a particular reason to use `Link` here since it's navigating off page, but doesn't matter, I think if we use `a` or `Link` tags. Not a change request, just pondering out loud. :D